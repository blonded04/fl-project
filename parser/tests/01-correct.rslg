START = lang;
lang = ["a"] + "ab" + {"o"} + "b" + ("a" | "us");