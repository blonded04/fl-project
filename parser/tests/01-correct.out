LexToken(START,'START',1,0)
LexToken(EQ,'=',1,6)
LexToken(NTERM,'lang',1,8)
LexToken(END_OF_LINE,';',1,12)
LexToken(NTERM,'lang',2,14)
LexToken(EQ,'=',2,19)
LexToken(LSB,'[',2,21)
LexToken(SYMBOL,'a',2,22)
LexToken(RSB,']',2,25)
LexToken(PLUS,'+',2,27)
LexToken(SYMBOL,'ab',2,29)
LexToken(PLUS,'+',2,34)
LexToken(LFB,'{',2,36)
LexToken(SYMBOL,'o',2,37)
LexToken(RFB,'}',2,40)
LexToken(PLUS,'+',2,42)
LexToken(SYMBOL,'b',2,44)
LexToken(PLUS,'+',2,48)
LexToken(LBR,'(',2,50)
LexToken(SYMBOL,'a',2,51)
LexToken(ALT,'|',2,55)
LexToken(SYMBOL,'us',2,57)
LexToken(RBR,')',2,61)
LexToken(END_OF_LINE,';',2,62)
